float64 rightFlap
float64 leftFlap
float64 rightElevator
float64 leftElevator
float64 onlyElevator
float64 altitudeRateTarget